core/src/main/java/org/apache/iceberg/BaseFileScanTask.java

```diff
@@ -30,7 +30,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 
-class BaseFileScanTask implements FileScanTask {
+public class BaseFileScanTask implements FileScanTask {
```

Comment:
This is made public so that unit test can directly construct FileScanTask.

Note that BaseCombinedScanTask is already public.